stylesheets/style.css

```diff
@@ -5,3 +5,185 @@ main-titles dark-grey: #2C303F;
 paragraph grey: #5b5e6d;
 nav grey: #5b5e6d;
 */
+
+/* ITERATION 1 */
+
+/*  Extra Small Screen (width < 768px */
+
+body {
+  margin: 0;
+  font-family: 'Heebo', sans-serif;
+  padding: 20px;
+}
+img {
+  max-width: 100%;
+}
+
+@media (max-width: 768px) {
```

Comment:
You don't need to put all your code inside this @mediaquery. Just specific things that only are different for that screen size. Please if you have questions regarding how to use media queries let us know.